src/record/record.js

```diff
@@ -249,17 +250,17 @@ Record.prototype.whenReady = function( callback ) {
  */
 Record.prototype._$onMessage = function( message ) {
 	if( message.action === C.ACTIONS.READ ) {
-		this._clearTimeouts();
+		clearTimeout( this._readTimeout );
```

Comment:
True, although ack timeout is whether or not we are subscribed to changes and response timeout is the time to get the data from the server. I get what you mean by if we get the data read it already means subscribe should pass, but since packets are sent serially if we don't get a subscribe ack before it means something went wrong too.

Is it possible to ever receive a response timeout without a subscribe ack?